packages/common/src/services/wallet-client/WalletClient.ts

```diff
@@ -138,45 +132,16 @@ export class WalletClient {
       )
 
       // Note: At this point, the funds are sitting in Wormhole.
-      // If the following code to redeem the funds on Solana fails,
-      // the funds can also get redeemed either via our background script on the
-      // recurring-jobs box or manually via the Wormhole Portal UI
-
-      // Initialize Wormhole TokenBridge client for redemption on Solana...
-      const wh = await wormhole('Mainnet', [evm, solana])
-      const sendChain = wh.getChain('Ethereum')
-      const receiveChain = wh.getChain('Solana')
-      const tokenBridge = await receiveChain.getTokenBridge()
-
-      const [whm] = await sendChain.parseTransaction(transferTxHash)
-      const vaa = await wh.getVaa(whm, 'TokenBridge:Transfer', 60_000)
-      if (!vaa) {
-        throw new Error('Could not get VAA from Wormhole to complete transfer.')
-      }
-
-      // Iterate through async generator of each Wormhole redemption tx and relay
-      const redeemTxs = tokenBridge.redeem(
-        new SolanaAddress(
-          (await sdk.services.solanaClient.getFeePayer()).toBase58()
-        ),
-        vaa
-      )
-      let wormholeTxIterator = await redeemTxs.next()
-      while (!wormholeTxIterator.done) {
-        console.debug('Sending redemption transaction...')
-        const tx = wormholeTxIterator.value.transaction
-        // TODO: Add priority fees
-        const signature = await sdk.services.solanaClient.sendTransaction(tx)
-        console.debug('Transaction sent successfully.', { signature })
-        wormholeTxIterator = await redeemTxs.next()
-      }
+      // The recurring jobs box will index the AudiusWormhole contract and
+      // auto-redeem the transfer.
 
       await pollForTokenBalanceChange(sdk, {
+        commitment: 'finalized',
         tokenAccount: account?.address,
         initialBalance: account?.amount,
         mint: 'wAUDIO',
-        retryDelayMs: 5000,
-        maxRetryCount: 720 /* one hour */
+        retryDelayMs: 10000,
+        maxRetryCount: 360 /* 60 minutes */
```

Comment:
Do we actually expect anyone to leave the client open for an hour 😅

Reply:
Yeah this was a yikes for me too